tests/unittests/config/test_cc_snap.py

```diff
@@ -189,71 +187,35 @@ def test_run_commands_on_non_list_or_dict(self):
             str(context_manager.exception),
         )
 
-    def test_run_command_logs_commands_and_exit_codes_to_stderr(self):
-        """All exit codes are logged to stderr."""
-        outfile = self.tmp_path("output.log", dir=self.tmp)
 
-        cmd1 = 'echo "HI" >> %s' % outfile
-        cmd2 = "bogus command"
-        cmd3 = 'echo "MOM" >> %s' % outfile
-        commands = [cmd1, cmd2, cmd3]
+@pytest.mark.allow_all_subp
```

Comment:
Nice conversion to pytest, but [our docs](https://cloudinit.readthedocs.io/en/latest/development/testing.html#test-layout) still say that all unit tests should live in a class. I'd be fine removing that (and making an exception here if we remove it right away), but I've tried to keep to that rule until now.

Reply:
I'm fine with being flexible with that rule if no class grouping makes sense (otherwise it is just boilerplate), but for now I'll just toss them in a class.